app/views/home/index.html.erb

```diff
@@ -8,11 +8,16 @@
             <p id="home-headline">View most recent posts:</p>
         </div>
         <% if @articles.any? %>
+            <% article_num = 1 %>
             <% @articles.each do |article| %>
```

Comment:
Looks like you're tracking the index of the article manually.
There's a method available on Enumerable (`@articles` is an enumerable) called [each_with_index](http://ruby-doc.org/core-2.5.1/Enumerable.html#method-i-each_with_index) that can help you out here.

eg:
```
<% @articles.each_with_index do |article, article_num| %>
  ... do stuff
  # the index starts at 0, so you might use `article_num + 1` to make it human-readable
<% end %>

Reply:
Awesome!